processor/deltatocumulativeprocessor/testdata/notemporality-ignored/1.test

```diff
@@ -25,13 +25,6 @@ resourceMetrics:
               dataPoints:
                 - timeUnixNano: 1
                   asDouble: 1
-          - name: test.summary
```

Comment:
I don't think this test is relevant anymore as we started to process summary data points. 
I added new testdata/summary dir to unit test the new logic